real_robots/constants.py

```diff
@@ -12,7 +12,8 @@
 USING_ROBOBO = False
 USING_OMNIROBOT = False
 USING_OMNIROBOT_SIMULATOR = True
-assert (int(USING_REAL_BAXTER) + int(USING_ROBOBO) + int(USING_OMNIROBOT) + int(USING_OMNIROBOT_SIMULATOR)== 1), "You can only use one real robot at a time"
+assert (int(USING_REAL_BAXTER) + int(USING_ROBOBO) + int(USING_OMNIROBOT) + int(USING_OMNIROBOT_SIMULATOR) == 1), \
```

Comment:
you have a function in python called `sum()` for that kind of thing:

```python
assert sum([True, False, True] == 1
```
Also, it seems in that case that you cannot use anymore gazebo simulator, no? (i would check `<=1` then)